CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## [v0.6.0](https://github.com/Nexus-Mods/NexusMods.App/releases/tag/v0.6.0) - 2024-08-29
+
+# Caution: To update the app, you must completely uninstall the old version, including all mods. [Learn More.](https://nexus-mods.github.io/NexusMods.App/users/faq/#why-do-i-have-to-uninstall-everything-to-update-the-app)
+
+
+This release enhances support for Cyberpunk 2077, adds multiple Loadouts per game and implements the back-end changes to support our new "Loadout items" data model. 
```

Comment:
```suggestion
This release enhances support for Cyberpunk 2077, adds multiple Loadouts per game, and implements the back-end changes to support our new "Loadout items" data model.
```

Reply:
Thanks for the suggestions.

The "[Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)" is not required in English. It's a consistency thing, I don't use it anywhere else so it may not make sense to have it only in the changelog.